pkg/chunk/schema_config.go

```diff
@@ -28,6 +28,8 @@ var (
 	errInvalidSchemaVersion = errors.New("invalid schema version")
 	errInvalidTablePeriod   = errors.New("the table period must be a multiple of 24h (1h for schema v1)")
 	errConfigFileNotSet     = errors.New("schema config file needs to be set")
+	errConfigChunkNotSet    = errors.New("schema config for chunks is missing")
+	errConfigIndexNotSet    = errors.New("schema config for index is missing")
```

Comment:
I would be more clear.

```suggestion
	errConfigIndexPrefixNotSet    = errors.New("schema config for index is missing the 'prefix' setting")
```